sympy/sets/sets.py

```diff
@@ -940,6 +941,15 @@ def right_open(self):
         """
         return self._args[3]
 
+    @property
+    def is_empty(self):
```

Comment:
This method is not needed. If the intersection is empty, `_intersect` will return `EmptySet` and that will have `is_empty = True`.

Reply:
The problem, however,@jksuom, is that the `_intersect` method is no more.
I think it will be good to proceed in this direction.

Reply:
The merge conflicts have also been corrected.